### PR TITLE
[4.0] sessionID & pgsql

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #33740 . alternative to #33787

### Summary of Changes
Handle resources

### Testing Instructions
see #33740
use postgresql
change session to file
login/logout

### Actual result BEFORE applying this Pull Request
error

### Expected result AFTER applying this Pull Request
logout successfully

### CODE REVIEW

File: plugins/user/joomla/joomla.php [hunk 1]
```diff
@@ -426,6 +426,15 @@ public function onUserLogout($user, $options = [])
 
 			/** @var SessionManager $sessionManager */
 			$sessionManager = Factory::getContainer()->get('session.manager');
+			
+			// Handle resource
```
Comment (richard67) [inline plugins/user/joomla/joomla.php]:
```suggestion

			// Handle resource
```
Make PHPCS in Drone happy.

**Comment by richard67** [comment]
Alternative PR: #33819 .

**Comment by richard67** [comment]
Added release blocker label as inherited from the issue.

**Comment by joomdonation** [comment]
@alikon Should we close this in favor of https://github.com/joomla/joomla-cms/pull/33819 ?

**Comment by alikon** [comment]
```
i would like to avoid to touch the session lib, as this issue is not related to the session, but feel free to close it
```

**Comment by alikon** [comment]
```
i would like to avoid to touch the session lib, as this issue is not related to the session, but feel free to close it
```

**Comment by alikon** [comment]
```
i would like to avoid to touch the session lib, as this issue is not related to the session, but feel free to close it
```

**Comment by richard67** [comment]
@alikon If we want to do it your way with this PR, you have to do the same here, too:

https://github.com/joomla/joomla-cms/blob/b2849e9bf37a3ddbd07193f3957574d319a5e991/plugins/user/joomla/joomla.php#L134

**Comment by alikon** [comment]
I don't mind mine,yours, etc , I know mine is a band-aid, should be
resolved on database side not on session

Il mer 12 mag 2021, 12:45 Richard Fath ***@***.***> ha
scritto:

**Comment by richard67** [comment]
@alikon Both are not nice and I understand what you mean. But as I said, please add the same change at the other place I've mentioned because we have the same issue there, reading the session id with loadColumn and so on.

**Comment by PhilETaylor** [comment]
I have PRed your branch @alikon for easy merging here https://github.com/alikon/joomla-cms/pull/71

**Comment by richard67** [comment]
@alikon If you merge @PhilETaylor 's PR https://github.com/alikon/joomla-cms/pull/71 , I'll close my PR in favour of this one.

**Comment by joomdonation** [comment]
I still prefer doing it in a central place but guess we will have to respect API design.

**Comment by alikon** [comment]
i don't know after this commit https://github.com/joomla/joomla-cms/commit/f8445efcc10260a453f14740194225060476bd1a
what is the current situation

**Comment by richard67** [comment]
I've closed my PR now in favour of this one.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on a6536fdddeee383a79524ccceada5b692c39da78<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33817">issues.joomla.org/tracker/joomla-cms/33817</a>.</sub>

**Comment by richard67** [comment]
@alikon Meanwhile I favour @nibra 's new PR #33822 . Please check if you come to the same conclusion.

**Comment by alikon** [comment]
closed in favour of #33822

**Pull request closed without merging**